python/desc/imsim/imSim.py

```diff
@@ -152,6 +152,12 @@ def metadata_from_file(file_name):
     return commands
 
 
+def uss_mem():
+    """Return unique set size memory of current process in GB."""
+    my_process = psutil.Process(os.getpid())
+    return my_process.memory_full_info().uss/1024.**3
```

Comment:
I did some research and it looks like this is also supposed to work on OSX:

http://www.pybloggers.com/2016/02/psutil-4-0-0-and-how-to-get-real-process-memory-and-environ-in-python/

But it might not be bad to double check this doesn't do something crazy if we aren't on Linux.

Reply:
No worries.  It works on OSX.  (My Mac at least.)  And it has both `rss` and `uss` attributes as required.